aries-site/src/pages/design-tokens/index.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {
+  Box,
+  Data,
+  DataContext,
+  DataSearch,
+  DataSummary,
+  Heading,
+  Grid,
+  PageContent,
+  Paragraph,
+  Notification,
+} from 'grommet';
+import { Meta, ContentCard } from '../../components';
+import { getCards, getPageDetails, nameToPath } from '../../utils';
+
+const title = 'Design tokens';
+const pageDetails = getPageDetails(title);
+const cards = getCards(title);
+
+const Tokens = () => (
+  <>
+    <Meta
+      title={title}
+      description={pageDetails.seoDescription}
+      canonicalUrl="https://design-system.hpe.design/tokens"
+    />
+    <PageContent>
+      <Box pad={{ vertical: 'medium' }} justify="center" width="large">
+        <Heading margin="none">{title}</Heading>
+        <Paragraph size="large">{pageDetails.description}</Paragraph>
+      </Box>
+      <Notification
+        message={`Design tokens are in a beta phase. This documentation is in 
+            progress and subject to change.`}
+        width="large"
+        margin={{ bottom: 'large' }}
+      />
+      <Data data={cards} pad={{ bottom: 'large' }}>
+        <DataSearch width={{ max: 'medium', width: '100%' }} />
```

Comment:
Is search necessary at this point in time with only 5 spokes?

Reply:
Don't think so, will remove.

Reply:
Done.